hbase-plugins/pom.xml

```diff
@@ -109,6 +109,14 @@
           <groupId>log4j</groupId>
           <artifactId>log4j</artifactId>
         </exclusion>
+        <exclusion>
```

Comment:
nit: If these conflict with other dependencies, can we add a brief comments here on which ones?